src/container/ModalDialog/ModalDialog.tsx

```diff
@@ -17,12 +17,12 @@ const ModalDialog: FC<ModalDialogProps> = (props) => {
   return (
     <ModalBackground>
       <div className='w-4/5 h-auto lg:w-1/2 xl:w-2/5 xl:max-w-screen-sm bg-white text-center rounded-lg flex flex-col mx-5 dark:bg-gray-800 dark:text-white'>
-        <div className='flex'>
+        <div className='flex pr-1'>
           <button onClick={props.close} className='flex items-end ml-auto h-7 w-7 mt-1'>
             <XIcon className='w-full h-full' />
           </button>
         </div>
-        <div className='px-5'>{props.children}</div>
+        <div className='px-5 pb-10'>{props.children}</div>
```

Comment:
I'm not sure about that padding

Looks a bit weird with the e.g. the change password dialog

![image](https://user-images.githubusercontent.com/9399034/125984557-ac30b9db-9104-4b60-9956-95c1611c8c48.png)